modules/TestUtils.js

```diff
@@ -88,7 +88,7 @@ export const objectContains = (object, value, compareValues) => {
     compareValues = isEqual
 
   return Object.keys(value).every(k => {
-    if (isObject(object[k])) {
+    if (isObject(object[k]) && Object.keys(value[k]).length > 0) {
```

Comment:
Actually I think that `expect({}).toInclude({})` should pass, as should `expect(anyObject).toInclude({})` also pass. I just think the code can early-return.

If you use `Object.getOwnPropertyNames` (which is not shimmable), please use a fallback so that `expect` will continue to work in ES3 browsers.